### DISCUSSION TITLE
Command `mkdocs build` does not generate a site folder?

### DISCUSSION THREAD

**Discussion by jonasbirkelof** [squidfunk/mkdocs-material#7182] [Q&A] [answered]
Am I missing something or has something changed with how the `mkdocs build` command now works? It used to build the site to a `site` folder with html files, but now it just takes the markdown files and put them it in a zip-file. 

According to the [MkDocs documentation](https://www.mkdocs.org/getting-started/#building-the-site), the `mkdocs build` command should generate the html files.

**Comment by kamilkrzyskow** [comment, marked as answer]
Hello @jonasbirkelof,
the behaviour of `mkdocs build` did not change lately. It would help to see what is in your `mkdocs.yml`, are you perhaps running the `info` plugin?

**Reply by jonasbirkelof** [reply]
Wow, I'm an idiot! I had the info plugin activated, as you suggested. Thank you!
